tasks/linux.sh

```diff
@@ -161,4 +113,45 @@ case "$available_manager" in
         cmd_status="$(yum_status "\{ \"old_version\": ${old_version}, \"version\": \"%{VERSION}-%{RELEASE}\" \}")"
         success "$cmd_status"
     esac
+    ;;
+
+  "zypper")
+    # Non-interactive and no color
+    options+=("-n")
+
+    # <package>-<version> is the syntax for installing a specific version in zypper
+    [[ $version ]] && name="${name}-${version}"
+
+    case "$action" in
+      "install")
+        zyp_cmd="in"
+        ;;
```

Comment:
Sorry for the nitpick, but for apt and yum the case statemnets keep the order of `status`, `install`, `uninstall`, `upgrade` reading top to bottom. I think it would help this script be more readable/orgainized if we follow that patter with zypper.

Reply:
Pedantry is welcome, I'll do this.